eq.yml

```diff
@@ -487,8 +493,8 @@ jobs:
             -var publisher_tag=$eq_publisher_tag \
             -var survey_launcher_tag=$go_launch_a_survey_tag \
             -var schema_validator_tag=$eq_schema_validator_tag
-
-  - task: Wait for Deployed version
+  - task: Wait for Deployed Survey Runner
+    timeout: 10m
```

Comment:
@jonnyshaw89 why are we adding the timeout? and what was the reasoning for 10 minutes

Reply:
If the deploy fails it will keep waiting forever. this way the job will fail in concourse eventually